src/app/actions.js

```diff
@@ -166,7 +166,11 @@ export function logOut(store) {
       // again.
       dispatch(connectAndStartPolling(store, bakery));
     }
-  };
+  }
+  // Define a name that won't be munged by the minifier to check
+  // against in the check-auth middleware.
+  logOut.NAME = "logOut";
```

Comment:
This is a workaround for CRA's munging of function names in its minifier. This was required so that we could get rid of `craco` while also keeping authentication checks on dispatched thunks.